suncal/models/googlecal.py

```diff
@@ -139,7 +142,7 @@ def create_sun_calendar(
 
     with build("calendar", "v3", credentials=creds) as service:
         calendar = {"summary": calendar_title, "timeZone": timezone}
-
+        # pylint: disable=maybe-no-member"
```

Comment:
why do you need this? What does this setting do?

Reply:
Unfortunately pylint flags the google API calls, giving a E1101. Giving the google the benefit of a doubt, this might be a pylint false positive, see also http://pylint-messages.wikidot.com/messages:e1101